plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusConnectorConfig.java

```diff
@@ -106,6 +107,20 @@ public PrometheusConnectorConfig setBearerTokenFile(File bearerTokenFile)
         return this;
     }
 
+    @MinDuration("1s")
+    public Duration getPrometheusReadTimeout()
+    {
+        return prometheusReadTimeout;
+    }
+
+    @Config("prometheus.read.timeout")
```

Comment:
For consistency with other connectors:
```suggestion
    @Config("prometheus.read-timeout")
```